pkgs/default.nix

```diff
@@ -22,24 +23,58 @@ rec {
   active-modules = import ./active-modules {
     inherit pkgs;
     inherit self;
+    inherit all-modules;
   };
 
   upgrade-maps = import ./upgrade-maps {
     inherit pkgs;
   };
 
-  bundle-locked = pkgs.callPackage ./bundle-locked { inherit revstring; };
-
   bundle-image = pkgs.callPackage ./bundle-image {
     inherit bundle-locked revstring;
     inherit active-modules upgrade-maps;
   };
 
   bundle-image-tarball = pkgs.callPackage ./bundle-image-tarball { inherit bundle-image revstring; };
 
-  bundle-squashfs = pkgs.callPackage ./bundle-squashfs {
-    inherit bundle-locked revstring;
-    inherit active-modules upgrade-maps;
+  bundle-locked-fn = { modulesLocks }: pkgs.callPackage ./bundle-locked {
+    inherit modulesLocks;
+    inherit revstring;
+  };
+
+  bundle-locked = bundle-locked-fn {
+    moduleIds = null;
```

Comment:
how does this work? the bundle-locked-fn says it wants 1 paramater called modulesLocks but this is passing moduleIds

Reply:
oops.